app/src/main/java/com/recommend_tour/MainActivity.kt

```diff
@@ -1,20 +1,45 @@
 package com.recommend_tour
 
+import android.content.BroadcastReceiver
+import android.content.Context
 import android.content.Intent
+import android.content.IntentFilter
 import android.os.Bundle
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
-import com.recommend_tour.service.ApiService
+import com.recommend_tour.service.TourDataUpdateService
+import com.recommend_tour.viewmodels.TourListViewModel
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
+class MainActivity : AppCompatActivity() {
 
-//@AndroidEntryPoint
-class MyActivity : AppCompatActivity() {
-//    private val tourService: TourService by viewModels()
+//    private val viewModel: TourListViewModel by viewModels()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_test)
+        setContentView(R.layout.intro_layout)
```

Comment:
```suggestion
        setContentView(R.layout.activity_main)
```
하나의 activity에서 setContentView를 여러번 호출하는것도 가능하나
권장되는 방식은 아닙니다. 왜 그럴까요? 
activity를 분리하시던가, fragment를 활용해보세요.